roles/teuthology/files/vps.yaml

```diff
@@ -0,0 +1,14 @@
+overrides:
```

Comment:
Remind me why this file can't be in ceph.git/qa/suites/ ?

Reply:
@zmc because the way we use it I believe per @liewegas ' initial suggestion is we add `vps.yaml` on every line in crontab that runs on `vps` nodes

Reply:
That doesn't really answer the question :-/